src/main/java/at/hannibal2/skyhanni/features/garden/farming/FarmingArmorDrops.kt

```diff
@@ -88,9 +93,50 @@ class FarmingArmorDrops {
     }
 
     private fun checkArmor() {
-        val armorPieces = InventoryUtils.getArmor()
+        val armorPieces = InventoryUtils.getArmor(1500)
             .mapNotNull { it?.getInternalName_old() }
             .count { armorPattern.matcher(it).matches() }
         hasArmor = armorPieces > 1
     }
+
+    @SubscribeEvent
+    fun onRepoReload(event: RepositoryReloadEvent) {
+        try {
+            armorDropJson = event.getConstant("ArmorDrops") ?: error("ArmorDrops not found in repo")
+
+        } catch (e: Exception) {
+            e.printStackTrace()
+            LorenzUtils.error("error in RepositoryReloadEvent")
+        }
+    }
+
+    companion object {
+        var armorDropJson = JsonObject()
+        private var currentArmorDropChance = 0.0
+        private var lastCalculationTime = SimpleTimeMark.farPast()
```

Comment:
Is there a reason to put this in a companion object, instead of turning the entire class into an object? This just seems like a pointless separation. Alternatively you could use a dedicated API class.